operator/src/test/java/oracle/kubernetes/operator/helpers/KubernetesTestSupport.java

```diff
@@ -607,6 +633,10 @@ private T withOptionalCreationTimeStamp(T resource) {
       return resource;
     }
 
+    private void preserveStatusIfSeparate(String name, T resource) {
+
```

Comment:
should this be removed?

Reply:
Lenny, I'm not understanding why this might need to be removed...

Reply:
Yes; thanks for noticing that.